clients/frontend-authorization/src/oidc/browser/BrowserAuthorizationClient.ts

```diff
@@ -287,8 +280,7 @@ export class BrowserAuthorizationClient extends BrowserAuthorizationBase<Browser
     if (this._accessToken)
       return this._accessToken;
     if (requestContext)
-      requestContext.enter();
-    throw new BentleyError(AuthStatus.Error, "Not signed in.", Logger.logError, FrontendAuthorizationClientLoggerCategory.Authorization);
+      throw new BentleyError(AuthStatus.Error, "Not signed in.", Logger.logError, FrontendAuthorizationClientLoggerCategory.Authorization);
```

Comment:
This line's scope changed and is now in an if statement, I don't think that is right

Reply:
Resolved in a merge